packages/frontend/styles/dds-icons.css

```diff
@@ -0,0 +1,1851 @@
+@charset "UTF-8";
+.dds__icon {
```

Comment:
@qooban are we okay with that naming convention, or should we try to adjust it? 🤔 
All selectors in this file are written in [BEM naming convention](https://en.bem.info/methodology/naming-convention/) which is a good thing, but it's not consistent with the way of operating on class names on our project

Because of that I wonder if we should adjust it or leave it as it is 🤔 
I know that we could e.g. make a dictionary, which we could use to simplify usage of those icons and to omit confusion about the naming convention, but still I'm wondering which direction seems to be optimal
WDYT? 

Maybe this questions should be addressed during the implementation, not while introducing this file to repository 🤔

Reply:
I copied this file from Dell Design Library (https://www.delldesignsystem.com/foundations/icons/). I can't find the repository to check how often it is updated if at all. But I think if it is being updated we should consider that we may need to update this file in the future if Dell updates it. Looking what lines were changed and refactor it every time may be time consuming. So maybe making an exception for being inconsistent here is worth it?

Reply:
Yeah, you're mentioning a valid point
As I'm also wondering about it, I'd wait for the @qooban voice on that subject

### Note
After deeper thought, I wouldn't be worried about updating stylesheet related to icons, as from current point of view we wouldn't need more icons that are presented in the stylesheet and there is no indication how often icons library is being modified (I've tried to get that information from [changelog page](https://www.delldesignsystem.com/updates/changelog/))
Looking from other libraries perspective, icons are one of the library parts which is not often updated, once it's being included into a specific library
So IMO we shouldn't be worried about the timeliness aspect 💪

Reply:
1. We will be moving towards Dell Design System AFAIK, but it's uncertain how we will do the switch and which parts will be affected, which parts will have to be rewritten. We haven't finished switch to new theme fully yet and another one is coming in, it adds to complexity even more 🙂 
2. RND-169 won't solve all our concerns about DDS switch as it's big topic of course, it should be treated only as first, small step.
3. If we wouldn't have that offline environment requirement (mentioned in [this comment](https://cloudifysource.atlassian.net/browse/RND-168?focusedCommentId=79534)), we would just import the file directly as is and not store it. Maybe that requirement will be dropped in new Dell world, who knows ;-) 

... so in this specific case I think we shouldn't invest more time in adjustments, I'm good with having this CSS as is. To address inconsistency concern, maybe we could have a simple function for getting DDS icons class names, so that these classes will be used directly only in one place? 🤔 To be placed either in this repo or in cloudify-stage (for the time being). It would also be reasonable to have a new type (if DDS is not already providing something) to narrow down icon-name argument type, not to have any string, but predefined list of strings.

Anyways, I'm adding another point to my discussions with Alex B. and Alex M. - how we will do switch to DDS?

Reply:
Sure, in that case let's leave it as it is 💪 
I'm resolving all discussions as they're requiring some modifications from the code perspective